long-ride-alerts/src/test/java/org/apache/flink/training/exercises/longrides/LongRidesUnitTest.java

```diff
@@ -104,6 +132,15 @@ public void shouldAlertOnWatermark() throws Exception {
         assertThat(harness.numEventTimeTimers()).isZero();
     }
 
+    private Long resultingRideId() {
+        ConcurrentLinkedQueue<Object> results = harness.getOutput();
+        assertThat(results.size())
+                .isEqualTo(1)
+                .withFailMessage("Expecting test to have exactly one result");
+        StreamRecord<Long> resultingRecord = (StreamRecord<Long>) results.toArray()[0];
```

Comment:
Converting to array is unnecessary:
```suggestion
        StreamRecord<Long> resultingRecord = (StreamRecord<Long>) results.element();
```